### ISSUE TITLE
Specification

### ISSUE THREAD

**Issue by eudoxia0** [LispLang/lisplang.github.io#10]
Parse the TeX sources of the spec and produce something better than the CLHS.

Spec parser: https://github.com/eudoxia0/cl-ansi-spec

**Comment by TeMPOraL** [comment]
I recall @phoe-krk working on a better version of CLHS; maybe you two could join forces?

**Comment by phoe-krk** [comment]
Yes - I succeeded in machine-parsing the files of the draft and turning them into simple wiki markup using a set of regexes, as @TeMPOraL pointed out above. They of course still need a lot of handiwork, but they're mostly done.

I say we should join forces to avoid the Lisp curse of everyone doing their own thing and to actually have someone to rely on while working on giving CL a proper and contemporary specification.

**Comment by britishempire** [comment]
I agree wholeheartedly with the suggestion to merge @phoe-krk's into the lisp-lang site. It seems to me that CL as a whole needs a much of a contemporary, professional and 'cared-for' look than it's currently getting.

**Comment by phoe-krk** [comment]
@britishempire thanks for reminding me that I need to work more on this. xD
I'll try to push a few more pages today.

**Comment by vermiculus** [comment]
Link to the actual CLUS repository for reference: https://github.com/phoe/clus-data

**Comment by phoe** [comment]
@vermiculus Ha, you reminded me of this issue!
Yes, CLUS is progressing fast. You might want to use its sources or link to the deployed version at http://phoe.tymoon.eu/clus/doku.php

**Comment by vermiculus** [comment]
@phoe 😄 by the way, would you mind chatting about [this suggestion](https://www.reddit.com/r/lisp/comments/5xcafp/common_lisp_ultraspec_the_glossary_is_operational/deje13k/)? I have POC working in Elisp (which may ultimately be better-suited for the text-processing required).

**Comment by phoe** [comment]
@vermiculus responded. Thanks!
